docs/README.md

```diff
@@ -1,42 +1,30 @@
-docs-superstaq: Home of documentation for Superstaq
+docs: Home of documentation for Superstaq
 ===================================================
-This repository contains materials that support the [Superstaq documentation site](https://docs-superstaq.readthedocs.io/).
+This repository contains materials that support the [Superstaq documentation site](https://superstaq.readthedocs.io/en/latest/).
 
-## How to build docs-superstaq locally
+## How to build the docs locally
 ### Setup your environment
 
-Clone the repository, set up your virtual environment, install requirements, and make sure submodules are up-to-date.
+Clone the repository, set up your virtual environment, and install requirements.
 
-    git clone git@github.com:Infleqtion/docs-superstaq.git
-    python3 -m venv docs-superstaq-env
-    source docs-superstaq-env/bin/activate
-    cd docs-superstaq
+    git clone git@github.com:Infleqtion/client-superstaq.git
+    python3 -m venv venv_superstaq
+    source venv_superstaq/bin/activate
+    cd client-superstaq/docs
     pip install -r requirements.txt
```

Comment:
this will install the pypi versions of the superstaq clients. instead (i think?) we want to install all the client packages locally, as we instruct [here](https://github.com/Infleqtion/client-superstaq/blob/main/docs/source/resources/developer_guide.rst#push-edits-and-make-a-pull-request-for-review)

Reply:
good catch, thanks!